docs/modules/ROOT/pages/create-account.adoc

```diff
@@ -63,7 +63,7 @@ After you set up your account, you'll want to keep it safe. Here are some recomm
 
 - *Keep private information secure:* Keep your password and any xref:developer.adoc[API keys] private.
 
-- *Enable multi-factor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multi-factor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
+- *Enable multifactor authentication:* Protect your account from anyone who may know your username and password. With xref:multi-factor-authentication.adoc[multifactor authentication], you'll need to provide a unique code that's generated on your device as well as your username and password to sign in.
```

Comment:
This is not incorrect, simply an editorial choice. We tend to follow Merriam Webster: https://www.merriam-webster.com/dictionary/multi-factor%20authentication